backend/routes/db.py

```diff
@@ -158,19 +164,18 @@ def get_cset_members_items(
     if not columns:
         columns = ['*']
         # columns = ['codeset_id', 'concept_id']
-
+    query = f"""
+        SELECT DISTINCT {', '.join(columns)}
+        FROM cset_members_items
+        WHERE codeset_id {sql_in(codeset_ids)}
+    """
     with get_db_connection() as con:
-        query = f"""
-            SELECT DISTINCT {', '.join(columns)}
-            FROM cset_members_items
-            WHERE codeset_id {sql_in(codeset_ids)}
-        """
+        # TODO: these 3 following lines don't do anything; get overridden by the next query. which query is correct?
```

Comment:
## Bugfix todo: `get_cset_members_items()`
@Sigfried these 3 following lines don't do anything; get overridden by the next query. which query is correct?